ui/app/templates/vault/cluster/secrets/backend/overview.hbs

```diff
@@ -32,26 +32,31 @@
   <div class="box is-fullwidth is-shadowless has-tall-padding">
     <div class="selectable-card-container {{if (and (eq this.model.connections 403) (eq this.model.roles 403)) 'one-card'}}">
       {{#if this.model.connectionCapabilities.canList}}
-        <SelectableCard
+        <OverviewCard
+          data-test-selectable-card="Connections"
           @cardTitle="Connections"
-          @total={{if (eq this.model.connections 404) 0 this.model.connections.length}}
           @subText="The total number of connections to external databases that you have access to."
           @actionText="Configure new"
           @actionTo="vault.cluster.secrets.backend.create-root"
-          @queryParam={{"connection"}}
-          @type="role"
-        />
+          @actionQuery={{hash itemType="connection"}}
+        >
+          <Hds::Text::Display class="has-top-padding-m" @tag="h2" @size="500">
+            {{format-number (if (eq this.model.connections 404) 0 this.model.connections.length)}}
+          </Hds::Text::Display>
+        </OverviewCard>
       {{/if}}
       {{#if (or this.model.roleCapabilities.canList this.model.staticRoleCapabilities.canList)}}
-        <SelectableCard
+        <OverviewCard
           @cardTitle="Roles"
-          @total={{if (eq this.model.roles 404) 0 this.model.roles.length}}
-          {{! TODO: Messaging needs massaging }}
-          @subText="The total number of roles that have been set up that you can list."
+          @subText="The total number of roles configured that you have permissions to list."
           @actionText="Create new"
           @actionTo="vault.cluster.secrets.backend.create-root"
-          @queryParam={{"role"}}
-        />
+          @actionQuery={{hash itemType="role"}}
+        >
+          <Hds::Text::Display class="has-top-padding-m" @tag="h2" @size="500">
```

Comment:
I've never used this component before! Is this `HDS::Text::Display` component typically used for text with heading tags and `HDS::Text::Code` for code blocks?

Reply:
That was my understanding. There's also `<Hds::Text::Body>` I followed this to determine which component and tags to use https://hds-website-hashicorp.vercel.app/foundations/typography?tab=code#font-styles

Reply:
Good to know - so any text we place in the tags should use this `Hds::Text` component?

Reply:
Not necessarily - we just didn't have an equivalent typography helper class so I used this as a stand-in